client/src/app/site/assignments/components/assignment-list/assignment-list.component.ts

```diff
@@ -110,7 +110,16 @@ export class AssignmentListComponent extends BaseListViewComponent<ViewAssignmen
             follow: [
                 {
                     idField: 'assignment_ids',
-                    follow: [SPEAKER_BUTTON_FOLLOW],
+                    follow: [
+                        SPEAKER_BUTTON_FOLLOW,
+                        {
+                            idField: 'candidate_ids',
+                            follow: [{ idField: 'user_id', fieldset: 'shortName' }]
+                        },
+                        {
+                            idField: 'poll_ids'
+                        }
+                    ],
```

Comment:
@FinnStutzenstein: These fields are only requested for PDF-generation. Do you know a nicer way to request those fields only on demand?

Reply:
I would restructure this:
- You can create an own request for the PDF. Than it is encapsulated and maybe is put into the pdf-document-provider-service of each pdf.
- The AU service needs a one-shot route: It is like opening the autoupdate, get the initial data and close it afterwards. This could be currently done by the client, but a seperate route by the server would be good. So the server can send the initial data and close the connection afterwards.

Reply:
But how to send a request and wait for the initial data in the client? When does the client know, that the AU-service sent data?

Reply:
If the AU service would provide a single-shot route, it is just a normal http request. I do not favor hacking this in the client.

Reply:
Then I will open an issue at the AU-service.

Reply:
Hmm, but I think we need in the client the logic "send AU-request -> wait for response -> close connection". I think, this feature would be more used than such a "single-shot"-route in the AU-service. The reason is, that we have to wait for the response...